vendor/github.com/golang/mock/CONTRIBUTORS

```diff
@@ -0,0 +1,37 @@
+# This is the official list of people who can contribute (and typically
```

Comment:
Out of curiosity: any reason to go with censoring Vs go modules?

Reply:
ah, that is a separate debate. Initially vendoring made our productization effort easier, but we could think about dropping it, but I would not block this PR with that decision.